internal/test/wmcb/wmcb_test.go

```diff
@@ -112,10 +112,14 @@ func TestWMCB(t *testing.T) {
 	for _, vm := range framework.WinVMs {
 		log.Printf("Testing VM: %s", vm.GetCredentials().InstanceId())
 		wVM := &wmcbVM{vm}
-		files := strings.Split(*filesToBeTransferred, ",")
-		for _, file := range files {
-			err := wVM.CopyFile(file, remoteDir)
-			require.NoError(t, err, "error copying %s to the Windows VM", file)
+		folders := strings.Split(*dirsToBeTransferred, ",")
+		// all the files in `/payload` directory will be transferred to `C:\Temp`
+		// for files not in `/payload`, new directory will be created in `C:\Temp` which resembles the linux folder path
```

Comment:
I don't understand this comment. For one, this comment will need to be updated every time the input list of directories changes. So you need to say something to the effect of `not in the list of directories`. Moreover what do you mean by `for files not in /payload`?

>		// for files not in `/payload`, new directory will be created in `C:\Temp` which resembles the linux folder path

Where are we doing this? I think confusion is because you are using values instead of the referencing the variables here.

Reply:
Updated the comment to make it better to understand. LMK if its still confusing